### DISCUSSION TITLE
Thunderbolt doesn't recognise the antelope Audio Interface

### DISCUSSION THREAD

**Discussion by SashonIT** [osy86/HaC-Mini#1067] [Q&A] [answered]
Hello dear OSY

I have MAC OSX 13.2.1 running on the hades Canyon

When I connect the Antelope Orion Studio Synergy Core to the Thunderbolt port, it shows up in IOREG and System Preferences.  But then I can't control it with the Antelope Launcher. so the interface software doesn't register the interface.

What can I do in this case?

Someone flashed me the Thunderbolt controller years ago.

It was running and proofed under macOS Catalina 
I did a BIOS update from HN0067 to HN0070 before the Ventura installation.  Does this cause the firmware on the flashed Thunderbolt controller to go?
![Bildschirm­foto 2023-03-16 um 21 01 02](https://user-images.githubusercontent.com/126977959/225739404-f0bfcbb3-aa9e-47e5-97a7-5bcaf7fbd801.png)
<img width="1200" alt="Bildschirm­foto 2023-03-16 um 21 02 54" src="https://user-images.githubusercontent.com/126977959/225739828-e152cdaf-cd33-47aa-8bec-05b4614039a4.png">

Where can I find the Apple firmware to flash again?

Is it possible from macOS to read the firmware that is on the controller?

Can you recommend a SPI Flacher?

I am very grateful for tips and advice

Nice Greetings from Berlin / Germany

**Comment by osy** [comment]
From your screenshot, it seems like it’s loaded fine from TB’s perspective. Any issue is on the PCIe side. Are you sure the drivers are installed correctly? Does this work on a MacBook?

**Reply by SashonIT** [reply]
Oh Thank you for your Fast Reply.

Not tested yet.

Can do a Test on a M1Max Macbook at the weekend.

I also did a Workaround which was suggested by Antelope Audio.

Nothing Helped until now.

Update.  Now nothing is recognizing.
<img width="720" alt="Bildschirm­foto 2023-03-16 um 22 45 25" src="https://user-images.githubusercontent.com/126977959/225759535-307cc7dc-585a-47bc-9fe1-345e5b7f1bca.png">




Where do i get the Firmware to Flash?

**Comment by SashonIT** [comment, marked as answer]
Thunderbolt was running but not with the Antelope Audio Orion Studio Synergy Core.

Now it is.

Because of this.
https://www.hackintosh-forum.de/forum/thread/47303-probleme-mit-virtualbox-auf-dem-hackintosh/?postID=710720#post710720

Yes it is in German.  But we all have Translation- Tools running.

Also Chat-GPT can do this.

I am super happy now.

Thank you OSY for this Repo and Project.

We are Brothers in mind for the decision to make a better MAC out of this Crazy Machine, called NUC8I7HVK.
This Sentences could be from Apple in one of their Keynotes :-)).  HAHAHAHAHA

Thanks for all.

Wish you all the best!

**Reply by ivan-ef** [reply]
> Because of this. 

Yes, disable IO mapper also works for other thunderbolt devices like ethernet controllers, etc. I think I didn't play with VT-d option at all.

**Reply by SashonIT** [reply]
Thanks for the Hint.

Everything is working now.
Antelope Audio Interfaces really need VT-D enabled in MacOS 13.2.1.

Only Airdrop is a One-Way Thing at the Moment. But soon solved. I ordered a different Wireless/Bluetooth Card https://www.ebay.de/itm/392844222499.  BCM94360NG Hackintosh Wifi BCM94360NG M.2 Wireless-Karte mit I-PEX4 WiFi-Antenne
which works OOB
Cause Ventura dropped the support of the Dell dw1820 which i have in.

Airdrop recieving is only possible not sending.

---> https://osxlatitude.com/forums/topic/11138-inventory-of-supportedunsupported-wireless-cards-2-sierra-ventura/?tab=comments#comment-86614


Anway.

I will write down my way in a complete Documentation and post here anyway.

Was a tough way.

**Reply by topiga** [reply]
You could also try this [SSDT-DMAR-DMAC.zip](https://github.com/osy/HaC-Mini/files/11129233/SSDT-DMAR-DMAC.zip) instead. Add this to your config and disable "disableIOMapper"

**Comment by ivan-ef** [comment]
> Only Airdrop is a One-Way Thing at the Moment. But soon solved. I ordered a different Wireless/Bluetooth Card https://www.ebay.de/itm/392844222499. BCM94360NG Hackintosh Wifi BCM94360NG 

For everyone who will read this post: Only Apple wifi module will support airdrop since Monterey. BCM94360CS/2 or BCM943602CS are what you need. Not NG, not BAED or anything else, because brands other than Apple has their own device and vendor ID and not fully compatible with MacOS.
